cabal-helper.cabal

```diff
@@ -56,16 +56,22 @@ library
                        Paths_cabal_helper
   ghc-options:         -Wall
   build-depends:       base          < 5    && >= 4.5
-                     , Cabal         < 2.1  && >= 2.0     || < 1.26 && >= 1.14
+  if os(windows)
+    build-depends:     base          < 5    && >= 4.7
```

Comment:
Please leave off the lower bounds on the `os(windows)` case, they are redundant. All bounds on a package are essentially `&&`ed together so we end up with `< 5 && >= 4.5 && >= 4.7` which resolves to just `< 5 && >= 4.7` but without mentioning the `< 5` twice.